curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/6434759f78ec812264ff8f34.md

```diff
@@ -49,6 +49,12 @@ Your callback function should be empty.
 assert.match(code, /letters\.forEach\(\s*\(?\s*letter\s*\)?\s*=>\s*\{\s*\}/)
 ```
 
+Your callback function should be nested in `range(1, 99)`'s forEach callback
```

Comment:
In an effort to simplify this, we should explain more with code examples here like this

```suggestion
Your `letters.forEach()` callback function should be nested inside the `range(1, 99).forEach(number => {}` callback function.
```